views/js/qtiCreator/widgets/interactions/textEntryInteraction/states/Correct.js

```diff
@@ -108,11 +114,6 @@ define([
                     }
                     break;
                 case 'string':
-                    responseValue = convertedValue;
-                    if (responseValue === '') {
-                        widget.isValid(widget.serial, false, __('Invalid value in correct response property.'));
-                        return setErrorNotification($submitButton, element, attributes.baseType);
-                    }
```

Comment:
Now we don't have any validation for strings. Can we at least have something like `typeof responseValue === 'string' ` or `/^[a-zA-Z0-9]+$/.test(responseValue);` ?